### PR TITLE
Js json

### PR DESCRIPTION
**Pull request by GuhaPritam**

### CODE REVIEW

File: student-marks-board/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	// Parse data as js
+	.then((response) => response.json())
+	.then((data) => {
+		const standardOneData = data['standard-1'];
+		const standardTwoData = data['standard-2'];
+		const standardThreeData = data['standard-3'];
+		const standardFourData = data['standard-4'];
+		const standardFiveData = data['standard-5'];
+
+		reuse(standardOneData, standardOne, standardTwoData, standardTwo, standardThreeData, standardThree,
```
Comment (green-roots) [inline student-marks-board/index.js]: It is not a good practice to pass these many parameters.

**Review by green-roots — CHANGES_REQUESTED** [review]

File: student-marks-board/index.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+// Select standard one table
+const results = document.getElementById('results');
+
+// Fetch data from json
+fetch('./data/result.json')
+	// Parse data as js
+	.then((response) => response.json())
+	.then((data) => {
+		// renderOnScreen(data);
```
Comment (atapas) [inline student-marks-board/index.js:9]: Why is it commented?

File: student-marks-board/index.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+	fetch('./data/result.json')
+		// Parse data as js
+		.then((response) => response.json())
+		.then((data) => {
+			const newData = { ...data };
+
+			const standards = Object.keys(data);
+
+			standards.forEach((item) => {
+				newData[item] = data[item].filter((item) => item.name === searchName);
```
Comment (atapas) [inline student-marks-board/index.js:75]: Explain this line. Have you copied it from anywhere?

File: student-marks-board/index.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+function submitHandler(event) {
+	event.preventDefault();
+	const searchName = document.getElementById('button').value;
+
+	fetch('./data/result.json')
+		// Parse data as js
+		.then((response) => response.json())
+		.then((data) => {
+			const newData = { ...data };
```
Comment (atapas) [inline student-marks-board/index.js:70]: Explain this line. What is happening here?

**Review by atapas — CHANGES_REQUESTED** [review]

File: student-marks-board/index.js (hunk 2 above)
Comment (GuhaPritam) [reply student-marks-board/index.js:9]: for step by step i understood

File: student-marks-board/index.js (hunk 3 above)
Comment (GuhaPritam) [reply student-marks-board/index.js:75]: item is a function name

File: student-marks-board/index.js (hunk 4 above)
Comment (GuhaPritam) [reply student-marks-board/index.js:70]: newData is a variable

**Pull request closed without merging**